fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -140,7 +146,7 @@ def _eval(self, Q, lambda_vec):
             L_high = error + self.B * max_constraint
         return L, L_high, gamma, error
 
-    def eval_gap(self, Q, lambda_hat, nu):
+    def eval_gap(self, Q, lambda_hat, nu) -> "_GapResult":
```

Comment:
This should work without quotes since you're importing annotations, right? Maybe I'm missing something...

Reply:
You're absolutely right :+1: